src/main/java/vazkii/botania/common/item/equipment/bauble/ItemBauble.java

```diff
@@ -151,17 +142,16 @@ public void onEquipped(ItemStack stack, EntityLivingBase player) {
 				((EntityPlayer) player).addStat(ModAchievements.baubleWear, 1);
 
 			onEquippedOrLoadedIntoWorld(stack, player);
-			applyToPlayer(getBaubleUUID(stack), player.getUniqueID(), player.worldObj.isRemote);
 		}
 	}
 
-	public void onEquippedOrLoadedIntoWorld(ItemStack stack, EntityLivingBase player) {
+	@Override
+	public void onUnequipped(ItemStack stack, EntityLivingBase player) {
 		// NO-OP
 	}
 
-	@Override
-	public void onUnequipped(ItemStack stack, EntityLivingBase player) {
-		toRemoveItems.add(getBaubleUUID(stack));
+	public void onEquippedOrLoadedIntoWorld(ItemStack stack, EntityLivingBase player) {
+		// NO-OP
```

Comment:
This method was added, but doesn't appear to have any usages. Please remove if it's no longer necessary.

Reply:
`onEquippedOrLoadedIntoWorld` is still used, it's just a no-op in ItemBauble, it's called and overriden.